doc/api/v1.yaml

```diff
@@ -46,23 +46,19 @@ paths:
           description: One of messages is malformed
 
   /api/v1/fragments/statuses:
-    post:
+    get:
       description: Get statuses of fragments
       operationId: FragmentStatuses
       tags:
         - fragment
-      requestBody:
-        description: List of fragment IDs
-        required: true
-        content:
-          application/json:
-            schema:
-              type: array
-              items:
-                type: string
-                pattern: '[0-9a-f]+'
-            example: ["68dcc12fe0dfe5e7b66ca6f8c959f9aa43b273e120a77fc3e4e2f04f1ecd7968"]
-          
+      parameters:
+        - in: path
+          name: fragment_ids
+          description: Comma-separated list of fragment IDs
+          schema:
+            type: string
+            pattern: '[0-9a-f]+(,[0-9a-f]+)+'
```

Comment:
I wonder how common this format is in our API. What is an idiomatic way to pass an array of simple values in REST/OpenAPI?

Reply:
I am not sure we have it in other places. The problem is that it was the only way to pass an array of these as an argument. Initially I wanted to use repeated `fragment_id`, but I iirc it just did was not possible.

Reply:
Query string does not forbid repeating keys AFAIK (worked with PHP array args in 2001!), but this way is more compact.

Reply:
This was the problem with `warp`, not the standard restricting me from doing that.